content/docs/command-reference/exp/push.md

```diff
@@ -47,6 +47,10 @@ to see experiments in the remote.
 This command will also try to `dvc push` all <abbr>cached</abbr> data associated
 with the experiments to [remote storage], unless `--no-cache` is used.
 
+If `studio.token` config or `DVC_STUDIO_TOKEN` environment variable is set, DVC
+will notify Studio about new experiments, and display a Studio project URL to
+view experiments.
```

Comment:
Like mentioned elsewhere, let's add a link or explain how to get the token here.